api/services/projects.py

```diff
@@ -81,3 +84,23 @@ def create_project_allocation(self, allocation: BaseProjectAllocation, created_b
         self.db.commit()
         self.db.refresh(new_allocation)
         return new_allocation
+
+    def get_project_stats(
```

Comment:
I don't see an issue of enabling to query against Vacation project, the data may not be useful but I don't see an issue with it.

But one thing a bit related is that we need to account for the yearly planned holidays in the planning calculation, but this can be a bit complex, we should discuss it together.